source/assets/stylesheets/styles/_page_header.scss

```diff
@@ -49,3 +49,11 @@
     font-size: 2.4em;
   }
 }
+
+.page-header__subtitle {
+  @include responsive-sub-title;
```

Comment:
:+1: for making this responsive text, however it looks slightly odd with the other text content that isn't responsive either. Perhaps we could use the `responsive-text` mixin for those elements?